core-build/gulp-core-build/src/tasks/jest.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "Jest Task configuration",
+  "description": "Defines Jest task configuration",
+
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "ci": {
+      "description": "Indicate whether the test is running in a CI environment",
```

Comment:
>"Indicate whether the test is running in a CI environment", [](start = 21, length = 59)

Why?  What does this do?

Reply:
What's the difference?

---
In reply to: [144159619](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r144159619) [](ancestors = 144159619)